### ISSUE TITLE
Configurability through uri

### ISSUE THREAD

**Issue by damooo** [NoelDeMartin/umai#16]
If app can be configured through uri, by providing storage root, user's webid, etc, then that can serve as an api fo

**Comment by josephguillaume** [comment]
I think configurability of the resource to open would be most useful, for deep linking

Noel, you've already posted about linking to recipes elsewhere, which it would be useful to be able to do with just a URI prefix, e.g. https://my-app.com/recipes/?id= yields https://my-app.com/recipes/?id=https://my-pod.com/cookbook/pizza#it

With multiple recipe lists or cookbooks, it would be useful to specify which to open. This could probably be done with the same query param as for a recipe.

If this is possible, then I think specifying the storage root would not be necessary - if a user has multiple storage roots or wishes to open a cookbook not in their storage root, they would already be able to do so.

I see two use cases for specifying webid in the Uri:
1) being able to derive a storage root for a different user, e.g. I want to see someone else's cookbook. This relates to https://github.com/NoelDeMartin/umai/issues/25 My guess is that this is better implemented through the ability to specify a cookbook to open, and it would be the linking application's responsibility to discover the user's cookbook and deep link straight to it.
2) specifying the account to log in with. As long as Solid-OIDC is the dominant authentication mechanism, I'm not sure how useful this would be because user input is still required. The most that can be achieved is streamlined autocomplete rather than full automation? Efforts with FedCM might change this, but Umai would then need to be compatible with FedCM rather than providing an custom means of specifying a webid.

**Comment by NoelDeMartin** [comment]
Yeah, I'm still not sure about the storage root or webId params, but maybe I could implement a generic way of deep linking and the app would perform client-site redirection to the appropriate route. For example `app.com?view=https://myrecipes.com/ramen#it` would redirect to `app.com/recipes/ramen?id=https://myrecipes.com/ramen#it`, etc.
